app/enterprise/2.1.x/kong-for-kubernetes/install.md

```diff
@@ -11,7 +11,7 @@ Kong for Kubernetes Enterprise provides most Kong Enterprise plugins and runs wi
 
```

Comment:
There's a "coming soon" note at the start of this file as well - can you change it to the same phrasing you use in the deployment options topic? Alternatively, you could just remove it.